app/views/cars/search.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1 class="text-center my-4">Cars</h1>
```

Comment:
@Dhyey54 When are we using this functionality?

Reply:
We search then the search method of the cars controller is called and the page is viewed